app/components/questionnarie/Questionnarie.tsx

```diff
@@ -37,6 +39,29 @@ const Questionnaire = ({ questions, options, quizLogic }: QuestionnarieProps) =>
     }
   }
 
+  useEffect(() => {
+    const handleKeyDown = (event: KeyboardEvent) => {
+      if (event.target && (event.target as HTMLElement).tagName === "INPUT") {
+        switch (event.key) {
+          case "ArrowRight":
+            nextQuestion()
+            break
+          case "ArrowLeft":
+            prevQuestion()
+            break
+          default:
+            break
+        }
+      }
+    }
+
+    window.addEventListener("keydown", handleKeyDown)
+
+    return () => {
+      window.removeEventListener("keydown", handleKeyDown)
+    }
+  }, [currentQuestionIndex])
+
   const isCurrentQuestionAnswered = typeof formState[`question${currentQuestionIndex}`] === "string"
```

Comment:
Could you create a variable for to be more explanatory?
ex ---> const numberQuestion =`question${currentQuestionIndex}`